eclipse-collections-code-generator/src/main/resources/copyrightAndOthers2018.stg

```diff
@@ -0,0 +1,11 @@
+copyrightAndOthers2018() ::= <<
```

Comment:
This file is not necessary. copyrightAndOthers.stg already has updated year: https://github.com/eclipse/eclipse-collections/blob/master/eclipse-collections-code-generator/src/main/resources/copyrightAndOthers.stg